src/test/java/Task3package/Сhef.java

```diff
@@ -0,0 +1,46 @@
+package Task3package;
+
+import java.util.ArrayList;
+
+public class Сhef {
+    private String dName;
+    private String dColor;
+    private double dWeight;
+    private int dCalories;
```

Comment:
same here? what is this prefix `d` stands for?

Reply:
Oh , now variables are unused , i tried to use them in methods DeleteByName (---  dName) , prefix 'd' means DELETE =) 
I`ll clean them